Sources/Extensions/Widgets/DetailsTable/WidgetDetailsTable.swift

```diff
@@ -0,0 +1,32 @@
+import AppIntents
+import Intents
+import Shared
+import SwiftUI
+import WidgetKit
+
+@available(iOS 17, *)
+struct WidgetDetailsTable: Widget {
+    var body: some WidgetConfiguration {
+        AppIntentConfiguration(
+            kind: WidgetsKind.detailsTable.rawValue,
+            intent: WidgetDetailsTableAppIntent.self,
+            provider: WidgetDetailsTableAppIntentTimelineProvider()
+        ) { timelineEntry in
+            WidgetDetailsTableView(entry: timelineEntry)
+                .widgetBackground(Color.clear)
+        }
+        .contentMarginsDisabledIfAvailable()
+        // @todo Alter displayName
+        .configurationDisplayName("Sensors")
```

Comment:
I imagine that by "Alter" you mean localization, but just commenting as a reminder, just add the string to Localizable.strings and run the project, then access it through `L10n.Something.something`